opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -200,15 +209,110 @@ namespace cpgrid
             int success = 1;
             // communicate success to others
             ccobj_.broadcast(&success, 1, 0);
-        }
 
-        NNCMaps nnc_cells;
-        // Add PINCH NNCs.
-        for (const auto& [cell1, cell2] : minpv_result.nnc)
-            nnc_cells[PinchNNC].insert({cell1, cell2});
+            // Add PINCH NNCs.
+            std::vector<Opm::NNCdata> pinchedNNCs;
+
+            for (const auto& [cell1, cell2] : minpv_result.nnc) {
+                nnc_cells[PinchNNC].insert({cell1, cell2});
+
+                if (pinchOptionALL) {
+                    auto topIJK = ecl_grid.getIJK(cell1);
+                    auto bottomIJK = ecl_grid.getIJK(cell2);
+                    std::vector<double> trans_between(bottomIJK[2]-topIJK[2]);
+                    std::vector<std::size_t> cells_between;
+                    cells_between.reserve(trans_between.size());
+
+                    // first we need to calculate transmissibilities from permeability and geometry
+                    auto trans = trans_between.begin();
+                    auto bottom_cell_info = ecl_grid.getCellAndBottomCenterNormal(cell1);
+
+                    // Wecalculate the transmissibilty tran for each intersection between the
+                    // two active cells. For each intersection we need geometry information
+                    // (distance from cell center to face center, face area and normal) of the
+                    // the top (cell_top) and the bottom cell (cell_bottom).
+                    // we start at the top (top cell is actived) and iterate until the bottom
```

Comment:
actived->active?

Also: the word active may be confused with the usual active cell concept, but I think it means something else here?

Reply:
Done.

To me it is the same. Those two cells are active due to the GRID and EDIT section. (MinPVProcessor might just have made more inactive).